docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -105,105 +112,95 @@ significant changes in each minor release.
-as a new version when necessary.
+A feature freeze means that we delay new features and fixes that are not related to the release candidate so we can ship the new release as soon as possible.
 
-Note that while there are no guarantees for providing fixes for unmaintained
-releases of Ansible, there can sometimes be exceptions for critical issues.
 
-.. _GitHub: https://github.com/ansible/ansible
+Release candidates
+^^^^^^^^^^^^^^^^^^^
 
-.. _release_changelogs:
+We create at least one release candidate before each new major release of Ansible or ``ansible-core``. Release candidates allow the Ansible community to try out new features, test existing playbooks on the release candidate, and report bugs or issues they find.
 
-Changelogs
-^^^^^^^^^^^^
+Ansible and ``ansible-core`` tag the first release candidate (RC1) which is usually scheduled to last five business days. If no major bugs or issues are identified during this period, the release candidate becomes the final release.
 
-We generate changelogs based on fragments. Here is the generated changelog for 2.9_ as an example. When creating new features or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the module documentation.
+If there are major problems with the first candidate, the team and the community fix them and tag a second release candidate (RC2). This second candidate lasts for a shorter duration than the first. If no problems have been reported for an RC2 after two business days, the second release candidate becomes the final release.
 
-We've got :ref:`examples and instructions on creating changelog fragments <changelogs_how_to>` in the Community Guide.
+If there are major problems in RC2, the cycle begins again with another release candidate and repeats until the core maintainers agree that all major problems have been fixed.
 
 
-Release candidates
-^^^^^^^^^^^^^^^^^^^
+.. _development_and_stable_version_maintenance_workflow:
 
-Before a new release or version of Ansible or ``ansible-core`` can be
-done, it will typically go through a release candidate process.
+Development and stable version maintenance workflow
+----------------------------------------------------
 
-This provides the Ansible community the opportunity to test these releases and report
-bugs or issues they might come across.
+In between releases, the Ansible community develops and maintains ``ansible-core`` on GitHub_, with a workflow that looks like this:
 
-Ansible and ``ansible-core`` tag the first release candidate (``RC1``)
-which is usually scheduled to last five business days. The final release
-is done if no major bugs or issues are identified during this period.
+ * Developers add new features and bug fixes to the ``devel`` branch.
+ * Each new feature and each notable bug fix includes a changelog fragment describing the work.
```

Comment:
```suggestion
 * Each new feature and each bug fix includes a changelog fragment describing the work.
```
IMO even a code refactoring should include a fragment. It can help later in case of bugs introduced